pkg/expr/translate/translate.go

```diff
@@ -259,7 +259,7 @@ func (dc *dashConditionsJSON) GetNew(orgID int64) (*ngmodels.Condition, error) {
 	ccAlertQuery := ngmodels.AlertQuery{
 		RefID:         ccRefID,
 		Model:         exprModelJSON,
-		DatasourceUID: expr.DatasourceUID,
+		DatasourceUID: expr.OldDatasourceUID,
```

Comment:
Sorry probably too tired right now. Why `expr.OldDatasourceUID` and not `expr.DatasourceUID`

Reply:
@kylebrandt

Reply:
Alerting would need to change, and there would need to be a migration to change the saved type.

Translate isn't actually used, dead code, but is similar to the ualert migration that migrations legacy 8 alerting conditions to new unified alerts with condition. Unified alerts saved model uses the "old" UID. So not sure we want to get into changing that now since it may require another migraiton.